receiver/prometheusreceiver/internal/otlp_metricfamily.go

```diff
@@ -120,6 +138,12 @@ func (mf *metricFamilyPdata) getGroupKey(ls labels.Labels) string {
 	return dpgSignature(mf.labelKeysOrdered, ls)
 }
 
+func (mg *metricGroupPdata) sortPoints() {
```

Comment:
I don't think I can find any usage of this (sorry if I missed a collapsed large diff)

Reply:
It is used at the top of `toDistributionPoint()` and `toSummaryPoint()`.  They seem to be in collapsed parts of the diff.